### DISCUSSION TITLE
components 폴더 안에 subatoms 폴더 생성 및 BookTitle 컴포넌트 추가에 대한 논의

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#132] [Ideas]
책나무 컴포넌트에 들어갈 책 그림 컴포넌트(BookBlock 컴포넌트)를 현재 atom 단위로 놓고 쓰고 있는데, 그 BookBlock 안에 들어가는 책의 제목 요소를 BookTitle이라는 컴포넌트로 나누어 관리하고자 합니다. 이 경우, 책의 제목을 렌더링해야 하는 모든 부분에서 BookTitle 컴포넌트를 가져다가 쓸 수 있습니다.
 그리고 BookBlock 컴포넌트 안에 들어가는 제목 요소라서 이거는 subatom 단위로 관리해야 할 것 같습니다. 그래서 의견을 받고, 다들 동의하신다면 북나무 컴포넌트를 제작 중인 예은 님께 요청해서 BookTitle 컴포넌트를 커밋 및 PR할 때 subatoms 폴더에 넣어 PR해달라고 요청하려고 합니다. 다른 분들도 만약 subatoms 폴더가 만들어진다면 atom 단위 컴포넌트 내 요소를 컴포넌트화 할 때 이용하실 수 있을 것 같습니다.
 의견 주시면 내일 취합 후 반영하겠습니다. 감사합니다!

**Comment by yen815** [comment]
저는 좋습니다!

**Comment by DragonTiger92** [comment]
일단 따로 의견이 없는 걸로 봐서 전체적으로 동의하신다고 생각하고 진행해도 될 것 같아요. 예은 님께서는 subatoms 폴더 만들어서 BookTitle 컴포넌트 관련 파일 PR할 때 같이 보내주시면 될 것 같아요. 일단 discussion은 닫을게요. 혹시 보시고 나중에 문제 되는 부분 있으면 다시 열고 토론 재개하도록 하겠습니다!

**Reply by yen815** [reply]
넹!!
